distributions/openhab/src/main/resources/bin/karaf

```diff
@@ -197,10 +199,6 @@ run() {
     nodebug=false
     while [ "${1}" != "" ]; do
         case "${1}" in
-            'clean')
```

Comment:
Does this mean that "clean" isn't available anymore? Is this a change that Karaf 4.3.1 introduced? Many people who believe "cleaning the cache" solves most issues might be pretty disappointed...

Reply:
${KARAF_DATA} points to all of ${OPENHAB_USERDATA}. Running this command would typically break openHAB, so this is probably a good thing.

Reply:
It's this upstream change 

https://github.com/apache/karaf/commit/aa870306c5bca0f1dd84bf20199fa00d63443c0c#diff-e3e003cfbe43c1837ab85d8f50141dc3f1936658fe35fd50ae637a00a2ae28fc

I think the openhabian scripts have their own cleaning commands to only clean the cache and tmp dir.